Plugin.pm

```diff
@@ -1124,7 +1129,8 @@ sub _albumItem {
 
 	if ($albumName) {
 		$item->{line1} = $albumName;
-		$item->{line2} = $artist;
+		$item->{line2} = $artist . ($albumYear ? ' (' . $albumYear . ')' : '');
+		$item->{name} .= $albumYear ? "\n(" . $albumYear . ')' : '';
```

Comment:
Which UI would respect that line feed ("`\n`")? If it's always respected, I'd put the year without the brackets, as they look odd, alone on a single line:

```
Greatest Album  - Favorite Artist
(1999)
```

Reply:
As far as I can tell, only the native LMS skins use the 'name' field to display the album information. All the other clients use 'line1' and 'line2'. As mentioned on the forum thread, I used the brackets on the year to be consistent with how the LMS skins display the release year with library albums when the Settings->Interface option to do so is specified.